app/src/main/res/layout/fragment_habit_list.xml

```diff
@@ -24,10 +24,12 @@
         android:id="@+id/add_habit_fab"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintBottom_toBottomOf="parent"
+        android:layout_marginEnd="20dp"
+        android:layout_marginBottom="70dp"
+        android:backgroundTint="@color/habit_add_button"
         android:clickable="true"
         android:focusable="true"
-        android:layout_marginBottom="70dp"
-        android:layout_marginEnd="20dp" />
+        android:src="@android:drawable/ic_input_add"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent" />
```

Comment:
These changes add the **plus** icon that was missing from the floating action button